test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java

```diff
@@ -247,8 +236,9 @@ public void close() throws Exception {
         }
     }
 
-    SegmentStoreStarter startSegmentStore(StorageFactory storageFactory, BookKeeperLogFactory dataLogFactory) throws DurableDataLogException {
-        return new SegmentStoreStarter(storageFactory, dataLogFactory);
+    SegmentStoreStarter startSegmentStore(StorageFactory storageFactory, BookKeeperLogFactory dataLogFactory, int containerCount)
```

Comment:
Do we need this method ?
why not just call return `new SegmentStoreStarter(storageFactory, dataLogFactory, containerCount); ` directly where it is needed?

Reply:
Done.